lib/connectors/gitlab/connector.rb

```diff
@@ -79,6 +75,7 @@ def extract_projects
           next_page_link = @extractor.yield_projects_page(next_page_link) do |projects_chunk|
             projects = projects_chunk.map { |p| Connectors::GitLab::Adapter.to_es_document(:project, p) }
             @sink.ingest_multiple(projects)
+            @status[:index_document_count] += projects.count
```

Comment:
This is not ideal.

We should examine the response of bulk API and see what operations succeed and what fail. Given `@sink` is a list of `sink`s, different `sink`s may get different results. 

We can improve this later
cc @artem-shelkovnikov @mchernyavskaya

Reply:
```suggestion
            @status[:indexed_document_count] += projects.count
```